src/Components/CreateImageWizardV2/steps/Review/ReviewStep.tsx

```diff
@@ -0,0 +1,196 @@
+    setIsExpandedContent(isExpandedContent);
+  const onToggleRegistration = (isExpandedRegistration: boolean) =>
+    setIsExpandedRegistration(isExpandedRegistration);
+  const onToggleImageDetail = (isExpandedImageDetail: boolean) =>
+    setIsExpandedImageDetail(isExpandedImageDetail);
+  const onToggleOscapDetails = (isExpandedOscapDetail: boolean) =>
+    setIsExpandedOscapDetail(isExpandedOscapDetail);
+
+  return (
+    <>
+      <ExpandableSection
+        toggleContent={'Image output'}
+        onToggle={(_event, isExpandedImageOutput) =>
+          onToggleImageOutput(isExpandedImageOutput)
+        }
+        isExpanded={isExpandedImageOutput}
+        isIndented
+        data-testid="image-output-expandable"
+      >
+        <ImageOutputList />
+      </ExpandableSection>
+      <ExpandableSection
+        toggleContent={'Target environments'}
+        onToggle={(_event, isExpandedTargetEnvs) =>
+          onToggleTargetEnvs(isExpandedTargetEnvs)
+        }
+        isExpanded={isExpandedTargetEnvs}
+        isIndented
+        data-testid="target-environments-expandable"
+      >
+        {environments.includes('aws') && <TargetEnvAWSList />}
+        {environments.includes('gcp') && <TargetEnvGCPList />}
+        {environments.includes('azure') && <TargetEnvAzureList />}
+        {environments.includes('oci') && <TargetEnvOciList />}
+        {environments.includes('vsphere') && (
+          <TextContent>
+            <Text component={TextVariants.h3}>VMware vSphere (.vmdk)</Text>
+            <TargetEnvOtherList />
+          </TextContent>
+        )}
+        {environments.includes('vsphere-ova') && (
+          <TextContent>
+            <Text component={TextVariants.h3}>VMware vSphere (.ova)</Text>
+            <TargetEnvOtherList />
+          </TextContent>
+        )}
+        {environments.includes('guest-image') && (
+          <TextContent>
+            <Text component={TextVariants.h3}>
+              Virtualization - Guest image (.qcow2)
+            </Text>
+            <TargetEnvOtherList />
+          </TextContent>
+        )}
+        {environments.includes('image-installer') && (
+          <TextContent>
+            <Text component={TextVariants.h3}>
+              Bare metal - Installer (.iso)
+            </Text>
+            <TargetEnvOtherList />
+          </TextContent>
+        )}
+        {environments.includes('wsl') && (
+          <TextContent>
+            <Text component={TextVariants.h3}>
+              WSL - Windows Subsystem for Linux (.tar.gz)
+            </Text>
+            <TargetEnvOtherList />
+          </TextContent>
+        )}
+      </ExpandableSection>
+      <ExpandableSection
+        toggleContent={'File system configuration'}
+        onToggle={(_event, isExpandedFSC) => onToggleFSC(isExpandedFSC)}
+        isExpanded={isExpandedFSC}
+        isIndented
+        data-testid="file-system-configuration-expandable"
+      >
+        <FSCList />
+      </ExpandableSection>
```

Comment:
amm not sure regarding that, it will be strange to remove this extandable form the review, maybe its good to leave it empty for now, @lucasgarfield wdyt?

Reply:
If it's too much commented code (which it probably is thinking about the toggles) then let's leave it. :)